views/beer-detail.hbs

```diff
@@ -0,0 +1,5 @@
+<div>
+    {{#each beer}}
+        {{> beer}}
```

Comment:
in this situation, I would feel more comfortable if you names these somehow different things. I know what I am reading, but this can get confusing super fast.
Always code assuming that the next time you see this you have no idea what you are looking at :)